src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1281,4 +1281,16 @@
   <data name="CollectionsShouldImplementGenericInterfaceMultipleMessage" xml:space="preserve">
     <value>Type '{0}' directly or indirectly inherits '{1}' without implementing any of '{2}'. Publicly-visible types should implement the generic version to broaden usability.</value>
   </data>
+  <data name="MakeTypesInternalCodeFixTitle" xml:space="preserve">
+    <value>Make all public types internal</value>
+  </data>
+  <data name="MakeTypesInternalDescription" xml:space="preserve">
+    <value>Different than libraries, applications aren't referenced and thus all types can be made internal.</value>
+  </data>
+  <data name="MakeTypesInternalMessage" xml:space="preserve">
+    <value>Since applications' code isn't referenced, all types can be made internal</value>
```

Comment:
Message could use some improvement; I don't have that good version cc @gewarren 
```suggestion
    <value>Since application's code typically not referenced publicly, types can be made internal</value>
```